src/main/kotlin/Main.kt

```diff
@@ -328,6 +332,16 @@ fun showPlayerDayDesc(town: Town, playerPos: Int, messageId: Long, chatId: Long,
     }
 }
 
+fun getHideRolesMode(game: Game?): Boolean {
+    return game?.let { game ->
+        game.host?.settings?.hideRolesMode
+    } ?: false
+}
+
+fun getHideRolesMode(gameId: GameId): Boolean {
```

Comment:
What is this? if you have game everywhere, just use the first function, if you don't then use second one
But I don't see mush reason for any of it
Especially of the only reason is that you have town instead of a game at some places. Just pace a game and retrieve town from it instead.